### DISCUSSION TITLE
After upgrade to 3.0 some colors are missing.

### DISCUSSION THREAD

**Discussion by theoreaves** [tailwindlabs/tailwindcss#6831] [Help]
Hello,

I've upgraded tailwindcss to v 3.0.  I followed the guide and all is well, except I seem to be missing some colors that I'm using.  For example "bg-blue-500", I only see blue-100, 300 and 600.  What am I mising?

Here is my tailwind.config.js file:
```
// tailwind.config.js
const colors = require('tailwindcss/colors');
module.exports = {
    content: [
        './vendor/laravel/framework/src/Illuminate/Pagination/resources/views/*.blade.php',
        './vendor/laravel/jetstream/**/*.blade.php',
        './storage/framework/views/*.php',
        './resources/views/**/*.blade.php',
    ],

    theme: {
        extend: {
            colors: {
                'blue-tb1': '#3fa4dc',
                'blue-tb2': '#2c2c79',
                green: colors.emerald,
                yellow: colors.amber,
                purple: colors.violet,
                gray: colors.neutral,
            },
            height: {
                112: '33rem',
                'screen-60': '60vh',
            },
            width: {
                100: '28rem',
            },
            fontSize: {
                xsm: '.50rem',
            },
        },
    },

    plugins: [require('@tailwindcss/forms')],
};
```

**Comment by diahron** [comment]
In the docs check out: Color palette changes.  (https://tailwindcss.com/docs/upgrade-guide#removed-color-aliases)

**Reply by theoreaves** [reply]
So, I figured out that my issue is that I have several css styes that I set programmatically.  Some come from a database entry even.  So, the issue is that when tailwindcss v3 scans the source code, it doesn't find these values.  This wasn't an issue in v2.  Is there anyway to have it generate all the style values?  Don't get me wrong, I get why it is good to not include values that are not used by the program, but in my case, I really do need those values.  using a CDN really isn't an issue, because I've also added to those values as well.

**Reply by adamwathan** [reply]
You probably just want to safelist those classes:

https://tailwindcss.com/docs/content-configuration#safelisting-classes

**Reply by theoreaves** [reply]
Thanks!  That is exactly what I was looking for.
